### DISCUSSION TITLE
Possible to open output and switch back to previous window?

### DISCUSSION THREAD

**Discussion by Supreeeme** [mfussenegger/nvim-dap#361] [Q&A] [answered]
When I run the debugger, I want to run the program without the output screen stealing the focus. I was looking at the terminal_win_cmd option but it doesn't do what I want and I imagine this doesn't work because it seems like the output is just opened in the current buffer. Is there something I can do to accomplish this?

**Comment by mfussenegger** [comment, marked as answer]
Per default the integrated terminal shouldn't steel the focus. Can you clarify the behavior you are seeing and what you'd expect?

**Reply by Supreeeme** [reply]
Huh, I could've sworn it was stealing the focus, but I just tried it and it definitely doesn't lol. Whoops. Is there a way to set options on the window though? (i.e. set wrap or set winfixheight)

**Reply by mfussenegger** [reply]
Yes it is possible with `terminal_win_cmd`. It is a command executed with `vim.cmd`, so any commands you can use using command mode with `:` can be used.

Something like this should work:

```lua
lua require('dap').defaults.fallback.terminal_win_cmd = 'sp | new | set wrap'
```

But you can also write custom functions that set more options, see https://github.com/mfussenegger/nvim-dap/issues/214#issuecomment-860976263

**Reply by Supreeeme** [reply]
Nice, thanks!
